taxi/views.py

```diff
@@ -64,14 +66,36 @@ class CarDetailView(LoginRequiredMixin, generic.DetailView):
 
 class CarCreateView(LoginRequiredMixin, generic.CreateView):
     model = Car
-    fields = "__all__"
     success_url = reverse_lazy("taxi:car-list")
+    form_class = CarForm
 
 
 class CarUpdateView(LoginRequiredMixin, generic.UpdateView):
     model = Car
-    fields = "__all__"
     success_url = reverse_lazy("taxi:car-list")
+    form_class = CarForm
+
+
+def car_add_driver_view(request: HttpRequest,
+                        pk: int,
+                        driver_pk: int) -> HttpResponse:
+    car = Car.objects.get(pk=pk)
+    driver = Driver.objects.get(pk=driver_pk)
+
+    car.drivers.add(driver)
+
+    return HttpResponseRedirect(reverse("taxi:car-detail", args=(pk,)))
```

Comment:
You can use django redirect here(from django.shortcuts). It's not obligatory, but better practice.